runtime/src/lib.rs

```diff
@@ -84,8 +84,8 @@ impl sdk::Runtime for Runtime {
         if is_testnet() {
             // Testnet.
             Some(TrustRoot {
-                height: 9500687,
-                hash: "9cdd98b0acef04edf3056ce0b66b8323474f8c0048ad15ea6f97c3a4fc27a1d2".into(),
+                height: 11379296,
+                hash: "4a854864c82e3214bf560377d7a84650402ad0f5f4d2097f22f89757f95c6ade".into(),
```

Comment:
:information_source: that block on oasis scan https://testnet.oasisscan.com/blocks/11379296